### ISSUE TITLE
Executing a bash script dumps the env variables at the top of the window

### ISSUE THREAD

**Issue by mlcampbe** [swiftbar/SwiftBar#247]
I have a menu item like this:

echo "host.com | bash=ssh param1=userid@host.com terminal=true"

When I click this item it opens a terminal window as expected but the top of the window has:

export SWIFTBAR_VERSION='1.4' SWIFTBAR_PLUGIN_CACHE_PATH='/Users/<usrname>/Library/Caches/com.ameba.SwiftBar/Plugins/ssh.sh' SWIFTBAR_PLUGIN_PATH='/Users/<username>/tools/swiftBar-plugins/ssh.sh' SWIFTBAR='1' SWIFTBAR_BUILD='359' SWIFTBAR_PLUGINS_PATH='/Users/<username>/tools/swiftBar-plugins' OS_APPEARANCE='Light' OS_VERSION_MINOR='15' SWIFTBAR_PLUGIN_DATA_PATH='/Users/<username>/Library/Application Support/SwiftBar/Plugins/ssh.sh' OS_VERSION_PATCH='7' OS_VERSION_MAJOR='10';ssh userid@host.com

Is there any way to suppress the printing of the env variables that get set?

**Comment by melonamin** [comment]
No option to disable it at this moment. What is bothering you? Just the looks?

**Comment by mlcampbe** [comment]
Mainly the looks to be honest. I was going to move my dock folder icon that contains commonly connected ssh hosts to a swiftbar menu and was slightly bothered by the fact that when connecting it displayed the extra info. However, after working on it a bit I discovered that my ssh script is not working as I had expected. Yes, it presents my remote hosts and I can connect to them just fine but upon logout I have to logout of both the remote hosts and then logout of my local terminal session to close the window. That was not my expectation so I've abandoned the idea for now.
